esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -1901,10 +1901,24 @@ void CWsDfuEx::doGetFileDetails(IEspContext &context, IUserDescriptor* udesc, co
     }
     //#14280
 
-    FileDetails.setWuid(df->queryAttributes().queryProp("@workunit"));
+    const char* wuid = df->queryAttributes().queryProp("@workunit");
+    FileDetails.setWuid(wuid);
     if (version >= 1.28)
         FileDetails.setNumParts(df->numParts());
 
+    if (cluster && *cluster && wuid && *wuid && (version >= 1.31))
+    {
+        Owned<IDFUWorkUnitFactory> factory = getDFUWorkUnitFactory();
+        Owned<IDFUWorkUnit> wu = factory->updateWorkUnit(wuid);
```

Comment:
i.e. as it stands (apart from being expensive) it will have no effect for files created by the hThor/Thor/Roxie.